src/utils/chart-utils.tsx

```diff
@@ -0,0 +1,166 @@
+import { Box, Typography } from "@mui/material";
+import { TooltipProps } from "recharts";
+import { ValueType, NameType } from "recharts/types/component/DefaultTooltipContent";
+import { DailyEntry, PersonTotalTime } from "../generated/client";
+import strings from "../localization/strings";
+import { theme } from "../theme";
+import { getHoursAndMinutes } from "./time-utils";
+import { CustomLabel } from "../types";
+
+
+/**
+ * Reformats inputted daily entry to be presented in the pie chart
+ *
+ * @param dailyEntry
+ * @returns an array of elements each representing a section in the pie chart
+ */
+export const dailyEntryToChart = (dailyEntry: DailyEntry) => {
```

Comment:
You resolved the conversation but its still could be changed to an implicit return